mne/cov.py

```diff
@@ -154,6 +157,16 @@ def save(self, fname):
 
         end_file(fid)
 
+    def copy(self):
+        """Copy the Covariance object
+
+        Returns
+        -------
+        cov : instance of Covariance
+            The copied object.
+        """
+        return deepcopy(self)
```

Comment:
Sorry that's my bad. Feel free to remove. I think I added it because there
were a lot of copies being done.